### ISSUE TITLE
how to analyze the whole game

### ISSUE THREAD

**Issue by ShareW97** [featurecat/lizzie#743]
Is it possible for katago to analyze the whole game with a detailed report with many variations of every single move and overall ratings?

**Comment by featurecat** [comment]
Yep, you can press 'a' to automatically analyze the entire game. If you've enabled the detailed sgf saving, then saving the game will include the variations.
